controllers/tools.js

```diff
@@ -29,7 +38,7 @@ const addToolHandler = (req, res, next) => {
 	const context = req.originalUrl.split('/')[1];
 	const action = `/${context}/${req.params.courseId}/tools/add`;
 
-	api(req).get('/ltiTools', { qs: { isTemplate: true, isHidden: false } })
+	api(req, { version: VERSION }).get('/ltiTools', { qs: { isTemplate: true, isHidden: false } })
```

Comment:
for the v3 variant, the query should be moved into the backend. the cleanest would be once again a specialized endpoint for the templates.